node/src/main/scala/coop/rchain/node/runtime/Setup.scala

```diff
@@ -318,9 +318,10 @@ object Setup {
       )
       cacheTransactionAPI <- Transaction.cacheTransactionAPI(transactionAPI, rnodeStoreManager)
       webApi = {
-        implicit val (ec, bs, or, sp) = (engineCell, blockStore, oracle, span)
-        implicit val (ra, rc)         = (rpConfAsk, rpConnections)
-        val isNodeReadOnly            = conf.casper.validatorPrivateKey.isEmpty
+        implicit val (ec, bs, or, sp)      = (engineCell, blockStore, oracle, span)
+        implicit val (ra, rc)              = (rpConfAsk, rpConnections)
+        val isNodeReadOnly                 = conf.casper.validatorPrivateKey.isEmpty
+        implicit val rm: RuntimeManager[F] = runtimeManager
```

Comment:
I am ok with both implicit or explicit parameter here.(Personally, I am more fond of explicit here)

Reply:
Resolved in [commit](https://github.com/rchain/rchain/pull/3584/commits/0912026303b68c5a6c203b33e6a2cb9151c05d5c).